roles/k8s_best_practices_certsuite/tasks/teardown.yml

```diff
@@ -22,36 +22,17 @@
   ignore_errors: true
 
 # Deleting resources created by the certsuite to support the execution
+# Deleted tasks that are no longer needed,refer CILAB-2102
```

Comment:
Please don't include documentation that is not publicly available, from the previous line I think this line could be omitted.